tests/btrfs-progs/install.pm

```diff
@@ -17,7 +17,8 @@ use base 'opensusebasetest';
 use utils;
 use testapi;
 
-my $STATUS_LOG = '/opt/status.log';
+my $status_log = '/opt/status.log';
```

Comment:
I think you could remove the `package install;` statement from line [LINE:12](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7583/files#diff-7e7917db151bbfdb05ca582da82476c5R12). The `loadtest()` function will add a propper namespace for you see https://github.com/os-autoinst/os-autoinst/blob/master/autotest.pm#L109

Reply:
I agree. I have remove the package install. But the root cause is just my typo for STATUS_LOG :)